package/kernel/linux/modules/other.mk

```diff
@@ -802,6 +802,7 @@ $(eval $(call KernelPackage,ikconfig))
 define KernelPackage/zram
   SUBMENU:=$(OTHER_MENU)
   TITLE:=ZRAM
+  DEPENDS:=+LINUX_6_12:kmod-lib-lzo
```

Comment:
Sorry, but this needs to be fixed correctly. `zram` hasn't strictly depended on `lzo` in over [four years](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/drivers/block/zram?id=3d711a382735d2c34d3ba2075a5aa83a894f4a57) (and I broke that dependency precisely for the OpenWrt use case, in order not to bloat the kernel with unneeded code).

Reply:
In fact, due to this, new ZRAM_BACKEND_ settings have appeared in kernel 6.12, this patch needs to be rewritten.